### DISCUSSION TITLE
Clr angular to Core migration

### DISCUSSION THREAD

**Discussion by siddharth1903** [vmware/clarity#6274] [Q&A] [answered]
Team,

We are planning for a migration activity from angular to core and thought we'd get some help

- Is there some documentation planned around this that helps legacy consumers migrate. 
- some migration schematics that could automatically run on an existing code and migrate these tags? 
- if we are to run both together, how does coexistence look like in terms of branding and any other common stuff .. Like we have --clr-btn-primary-bg-color .. until a complete migration is done, we may have to look at maintaining theme variables corresponding to both core and angular?

**Comment by mathisscott** [comment]
CC: @bbogdanov

**Comment by bbogdanov** [comment, marked as answer]
Hi @siddharth1903,

The team is actively developing tools to enable you to switch from Clarity Angular to Clarity Core more easily and on your own pace. We are going to announce them soon.

There is also a website coming that will serve as a guidance on switching from using Clarity Angular components to Clarity Core components. The website will have docs on how to setup ESLint, visual side by side component comparison in Angular environment and code examples. 

On your last point - the team updated some of the colours in Clarity Core because of a11y. We are making sure every Core component we do is accessible. Having this, there are some visual colour changes that might disturb a page look and feel. We took an initiative to update `@clr/ui` to use the colours we use in `@cds/core` but I'm not sure what is the progress there so @mathisscott can give you info about where we are there.

I'll update the thread when we announce the adoption website and tools so you are aware when they come alive.

**Reply by siddharth1903** [reply]
Thanks @bbogdanov .. looking forward for the announcement ! :) We have planned for migration from the first week of october if things get approved internally... Do you think we'd get any help by that time?

**Reply by bbogdanov** [reply]
@siddharth1903 Our goal is to release the tools and the website till the end of October. From where we are at the moment I believe we are a good position to release the MVP earlier than that.
